packages/react/src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.jsx

```diff
@@ -352,7 +353,11 @@ class FilterHeaderRow extends Component {
           '--filter-header-dropdown-max-height': dropdownMaxHeight,
         }}
       >
-        {hasDragAndDrop && <TableHeader className={`${iotPrefix}--filter-header-row--header`} />}
+        {hasDragAndDrop && (
+          /* istanbul ignore next */ <TableHeader
+            className={`${iotPrefix}--filter-header-row--header`}
+          />
+        )}
```

Comment:
Please avoid `&&` when rendering elements. Use `hasDragAndDrop ? (<> ... </>) : null` instead. For more info, read [this article](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)